data/debadityasingh8972.json

```
@@ -0,0 +1,19 @@
+{
+  "name": "Debaditya Singh",
+  "displayStatsPublic": ture,
+  "type": "personal",
+  "bio": "Software Developer Fresher Looking for opportunities to test my skills | Full-stack Web Developer | Open-Source Contributor | Blockchain | Ethereum | DeFi | 3⭐ @LeetCode |"
+  "avatar": "https://github.com/debadityasingh8972.png",
```

We no longer need the `avatar`. `type`, & `displayStatsPublic` keys. Please remove these from your `json`! 

```suggestion
  "bio": "Software Developer Fresher Looking for opportunities to test my skills | Full-stack Web Developer | Open-Source Contributor | Blockchain | Ethereum | DeFi | 3⭐ @LeetCode |",
```